k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.ipp

```diff
@@ -100,8 +100,8 @@ namespace LCIO2EDM4hepConv {
       lval.setColorFlow(edm4hep::Vector2i(rval->getColorFlow()));
       lval.setVertex(edm4hep::Vector3d(rval->getVertex()));
       lval.setEndpoint(edm4hep::Vector3d(rval->getEndpoint()));
-      lval.setMomentum(Vector3fFrom(rval->getMomentum()));
-      lval.setMomentumAtEndpoint(Vector3fFrom(rval->getMomentumAtEndpoint()));
+      lval.setMomentum(rval->getMomentum());
```

Comment:
Can we also remove the `Vector3fFrom` now, or is that still in use in other places?

Reply:
Ah, it looks like there are more overloads. I think this one should be removable now:

https://github.com/key4hep/k4EDM4hep2LcioConv/blob/697a942654c7be363cec4ee63384585271ec63ef/k4EDM4hep2LcioConv/include/k4EDM4hep2LcioConv/k4Lcio2EDM4hepConv.h#L163

Reply:
It is actually not due to https://github.com/key4hep/EDM4hep/issues/265